modules/installation-configuration-parameters.adoc

```diff
@@ -629,7 +629,7 @@ endif::openshift-origin[]
 
 ifdef::aws[]
```

Comment:
Same comment about enterprise here. You would want to wrap the AWS condition in the same condition as above to prevent the new table row from appearing in the OKD doc.